.github/workflows/ci.yml

```diff
@@ -4,21 +4,13 @@ on:
   push:
     branches:
       - main
-      - main_0.1
-      - main_0.2
-      - main_0.3
-      - main_0.4
-      - main_0.6
+      - main_*
```

Comment:
Do we want it to actually run on main_*, or just main_0*

Reply:
Might as well be inclusive?